NDTensors/src/lib/TypeParameterAccessors/src/parameter.jl

```diff
@@ -0,0 +1,4 @@
+parameter(type::Type, pos) = parameter(type, position(type, pos))
+parameter(type::Type, pos::Position) = parameters(type)[Int(pos)]
+parameter(type::Type, pos::Int) = parameter(type, Position(pos))
```

Comment:
I don't think this function is necessary since you have this defined 
```
position(type::Type, pos::Int) = Position(pos)
```

Reply:
Fixed, thanks.